benchmarks/benchmark_serving.py

```diff
@@ -326,10 +479,17 @@ def main(args: argparse.Namespace):
     parser.add_argument(
         "--endpoint",
         type=str,
-        default="/generate",
+        default="/v1/completions",
         help="API endpoint.",
     )
-    parser.add_argument("--dataset",
+    parser.add_argument(
+        "--dataset-name",
+        type=str,
+        default="sharegpt",
+        choices=["sharegpt", "sonnet"],
```

Comment:
Would it make sense to, rather than getting dataset by name, specify whether we want to read from json or from a text file and provide a path to it? Or is it an overkill?

Reply:
Hmm - I think we can add the option to read from a json or path and that'll be nice for sure, but the problem is we'll have to use a single user base prompt for all datasets. Also certain arguments of the benchmark (e.g., configure input, output & prefix length) is only used for certain dataset. 

I haven't come up with a good solution so for now I keep them as separate datasets. I really like the idea of data registry where the way we sample prompts & output from the datasets lives outside the main benchmark script, and we can reuse it for other benchmark scripts as well, but I haven't put too much thoughts into it